components/accept-financial-contributions/AcceptContributionsMyself.js

```diff
@@ -0,0 +1,295 @@
+                          </Box>
+
+                          {error && (
+                            <Flex alignItems="center" justifyContent="center">
+                              <MessageBox type="error" withIcon mb={[1, 3]}>
+                                {error.replace('GraphQL error: ', 'Error: ')}
+                              </MessageBox>
+                            </Flex>
+                          )}
+
+                          <Flex justifyContent={'center'} mb={4}>
+                            <StyledButton
+                              fontSize="13px"
+                              minWidth={'85px'}
+                              minHeight="36px"
+                              type="button"
+                              onClick={() => {
+                                Router.pushRoute('accept-financial-contributions', {
+                                  slug: this.props.collective.slug,
+                                  path: this.props.router.query.path,
+                                }).then(() => window.scrollTo(0, 0));
+                              }}
+                            >
+                              {intl.formatMessage(this.messages.cancel)}
+                            </StyledButton>
+                            <StyledButton
+                              fontSize="13px"
+                              minWidth={'85px'}
+                              minHeight="36px"
+                              ml={2}
+                              buttonStyle="dark"
+                              type="submit"
+                              loading={this.state.loading}
+                              onSubmit={handleSubmit}
+                            >
+                              {intl.formatMessage(this.messages.save)}
+                            </StyledButton>
+                          </Flex>
+                        </Form>
+                      );
+                    }}
+                  </Formik>
+                </Box>
+              </Flex>
+              <Flex justifyContent="center" width={['100%', null, '60%', '50%']} my={[3, 0]}>
+                <FinancialContributionsFAQ width={['90%', '100%']} />
+              </Flex>
+            </Flex>
+          )}
+          {!router.query.method && <StripeOrBankAccountPicker />}
+        </Container>
+      </Fragment>
+    );
+  }
+}
+
+const bankAccountMutation = gqlV2`
+  mutation addBankAccount($account: AccountReferenceInput!, $key: AccountSettingsKey!) {
+    editAccountSetting(account: $account, key: $key, value: true) {
+      id
+      settings
+    }
+  }
+`;
+
+const addBankAccountMutation = graphql(bankAccountMutation, {
+  options: {
+    context: API_V2_CONTEXT,
+  },
+  props: ({ mutate }) => ({
+    addBankAccount: async ({ account, key, value }) => {
+      return await mutate({
+        variables: { account, key, value },
+        awaitRefetchQueries: true,
+        refetchQueries: [{ query: getLoggedInUserQuery }],
```

Comment:
Not that we removed refetchLoggedInUser these 2 lines are not needed anymore (awaitRefetchQueries, refetchQueries).

Note that I'm also refactoring that and even when we use `refetchLoggedInUser` we will not need that anymore in the future. https://github.com/opencollective/opencollective-frontend/pull/3990